scripts/ci/install.test

```diff
@@ -35,7 +35,8 @@ else
 fi
 
 # install the pre-built Bokeh package plus all test and runtime dependencies
-conda install -n testenv --yes --quiet --use-local bokeh `python scripts/deps.py run test` tornado=${TORNADO:-5}
+# use conda-forge channel only for installing test dependencies
+conda install -n testenv -c conda-forge --yes --quiet --use-local bokeh `python scripts/deps.py run test` tornado=${TORNADO:-5}
```

Comment:
For now I just want to test if 3.8 will work at all on CI (it works locally with this channel). However, in PR #9331 will need to install firefox and geckodriver, which are available only on conda-forge, so we will have set this up, this or another way, eventually.

Reply:
I think there is some way to specify channels to use for packages so perhaps we can limit C-F for just these two somehow

Reply:
I tried that (e.g. `conda-forge::firefox`) and it works with `conda install` but not with `conda build`.

Reply:
A longer syntax `'firefox [channel=conda-forge]'` (the quotes are needed) actually "works". By that I mean you have to run `conda build` like this `conda build -c conda-forge conda.recipe/`, which completely defeats the purpose of approach.

Reply:
Anyway, still doesn't work and takes conda only 40~50 minutes to tell why.

Reply:
Sophia said there were still packages to build/add, and hopefully this week or next? I would suggest pinging her later this week.

Reply:
> A longer syntax `'firefox [channel=conda-forge]'` (...) actually "works".

Well, that doesn't work as well, just fails later:
```
Error: Invalid selector in meta.yaml line 26:
offending line:
        - firefox [channel=conda-forge]
exception:
invalid syntax (<string>, line 1)
```